.github/workflows/rebase.yml

```diff
@@ -0,0 +1,15 @@
+name: Automatic Rebase
+
+on: [pull_request_target]
+
+jobs:
+  _:
+    name: "Automatic Rebase"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ljharb/rebase@master
```

Comment:
@ljharb what do you think about pinning this to a specific commit? It might be a volunerability on our side to rely on all new changes to this step in another repo. cc. @madicap

Reply:
That's totally fine if you prefer it; altho all of my projects don't bother pinning actions - there's no secrets to expose, altho i suppose pull_request_target in particular gives an action the ability to commit changes to the repo. However, branch protections would prevent it from doing so.

So, in summary, I don't think there's actually an attack vector here even if I went rogue and wanted to, but if you want them switched to a sha, go for it :-)